### ISSUE TITLE
Add API endpoint to update away status

### ISSUE THREAD

**Issue by vimtor** [calcom/cal.com#6882]
### Is your proposal related to a problem?

I wanted to add a new Raycast command to automatically update my away status from Raycast. The problem is that I did not found and endpoint in the docs to update this status.

### Describe the solution you'd like

I would want to have and endpoint and its corresponding docs to update my away status.

### Describe alternatives you've considered

I've considered implementing this myself but after searching through the code it didn't become clear what was the internal API and the external API. I stopped because maybe the `https://api.cal.com/` is not found in the code. I've seen that the away status is done using in the internal API with TRPC and the code is quite simple so maybe with a bit of guidance I can implement it myself for the external API.

**Comment by iprime2** [comment]
@vimtor is issue resolved?

**Comment by vimtor** [comment]
@iprime2 I don't think so, right? I was asking for an endpoint to update the away status, but I can't find it:

https://cal.com/docs/enterprise-features/api/api-reference#the-other-stuff

**Comment by ephraimduncan** [comment]
@PeerRich Can I look at this?

**Comment by CarinaWolli** [comment]
We remove the away status, so this is not needed anymore
